akka-docs/src/test/scala/tutorial_2/IotApp.scala

```diff
@@ -17,7 +17,7 @@ object IotApp {
 
     try {
       // Create top level supervisor
-      val supervisor = system.actorOf(IotSupervisor.props(), "iot-supervisor")
+      val supervisor = system.actorOf(IotSupervisor.props, "iot-supervisor")
```

Comment:
then the `props()` method in `object IotSupervisor` should also be changed. It can technically be a `val`. I think we often define them as `def` and with `()` because that is what is needed as soon as there are any parameters.

Reply:
well, looks like i've made a foolish mistake in my code in IotSupervisor. i'm so sorry about that. we should decline my issue and this pull request. you are totally right about props() and parentheses. Should i delete this issue and pull request by myself or it should be done by maintaners?

Reply:
no problem, contribution is welcome if you spot something else